profile/README.md

```diff
@@ -0,0 +1,10 @@
+# :zap: Unified Tech Power
+This organization is a combined effort of different project maintainers to provide a common framework for plugins from the "mod-like" genre.
+
+## :busts_in_silhouette: Who are we?
+This team consists of developers from the following projects, in alphabetical order.
+
+| Craftory | Nova | Slimefun | Space |
```

Comment:
You could link to the project's repository here and link to the organization/user at the bottom. (Maybe even clickable images?)

Reply:
The only issue with linking to projects is when one team represents more than one project. For example, Slimefun Org has both Slimefun4 and SensibleToolbox which are two separate projects.

Reply:
That's a good point.